woocommerce/woocommerce-paypal-payments
# Type error with advanced columns pro

```
Fatal error: Uncaught TypeError: WooCommercePayPalCommerceWcGatewayAdminOrderTablePaymentStatusColumn::render(): 
Argument #1 ($column) must be of type string, int given, called in 
.../wp-content/plugins/woocommerce-paypal-payments/modules/ppcp-wc-gateway/src/WCGatewayModule.php 
on line 618
```

### Steps to Test

1. Activate Paypal payments and advanced columns pro
2. See error
